server.go

```diff
@@ -1287,12 +1289,14 @@ func (s *Server) processUnaryRPC(t transport.ServerTransport, stream *transport.
 	// NOTE: this needs to be ahead of all handling, https://github.com/grpc/grpc-go/issues/686.
 	if s.opts.cp != nil {
 		cp = s.opts.cp
-		stream.SetSendCompress(cp.Type())
+		_ = stream.SetSendCompress(cp.Type())
```

Comment:
If we are guaranteed that `stream.SetSendCompress` will not return an error here, maybe replace this with:

```golang
// Safe to ignore returned error value as we are guaranteed to succeed here.
stream.SetSendCompress(cp.Type())
```

Same applies for other places where the return value from `SetSendCompress` is assigned to `_`.